nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -107,51 +106,9 @@ export default function GatewayConfig() {
     );
   }
 
-  function editGateway() {
-    return (
-      <AddEditGatewayButton
-        title={'Edit'}
-        isLink={true}
-        editProps={{
-          editTable: 'info',
-        }}
-      />
-    );
-  }
-
-  function editDynamicServices() {
+  function editFilter(editTab: EditProps) {
     return (
-      <AddEditGatewayButton
-        title={'Edit'}
-        isLink={true}
-        editProps={{
-          editTable: 'aggregation',
-        }}
-      />
-    );
-  }
-
-  function editEPC() {
-    return (
-      <AddEditGatewayButton
-        title={'Edit'}
-        isLink={true}
-        editProps={{
-          editTable: 'epc',
-        }}
-      />
-    );
-  }
-
-  function editRan() {
-    return (
-      <AddEditGatewayButton
-        title={'Edit'}
-        isLink={true}
-        editProps={{
-          editTable: 'ran',
-        }}
-      />
+      <AddEditGatewayButton title={'Edit'} isLink={true} editProps={editTab} />
```

Comment:
Actually, is this `isLink` prop still doing anything? It looks like we could remove it

Reply:
Yes it does, if it is set to false it is a button an not a link